src/open_inwoner/scss/components/TabPanel/TabPanel.scss

```diff
@@ -109,7 +109,7 @@
 
     // Styling for Benefits reports/Mijn uitkeringen
     .form {
-      //max-width: var(--mobile-ms-width);
+      max-width: var(--mobile-ms-width);
```

Comment:
Weet je het zeker dat deze mee moet komen in de patch?

Reply:
Ja! Anders zweeft het dropdown icoontje in ledige ruimte.